libyear/libyear

```diff
@@ -18,14 +18,15 @@ def main():
     pt = PrettyTable()
     pt.field_names = ['Library', 'Current Version', 'Latest Version', 'Libyears behind']
     total_days = 0
+    total_releases = 0
     for req in requirements:
         name, version, version_lt = get_requirement_name_and_version(req)
         if not name:
             continue
 
         if not version and not version_lt:
             continue
-
+        total_releases+=get_no_of_releases(name, version)
```

Comment:
Either add a command-line argument or remove the changes from this file for now.

Reply:
Yes, I have removed the changes.